pages/in-vitro-systems/[id].js

```diff
@@ -135,6 +143,12 @@ export async function getServerSideProps({ params, req }) {
           filterErrors: true,
         })
       : [];
+    const pooledFrom = inVitroSystem.pooled_from
+      ? await request.getMultipleObjects(inVitroSystem.pooled_from, null, {
+          filterErrors: true,
+        })
+      : [];
+    const partOf = await request.getObject(inVitroSystem.part_of, null);
```

Comment:
It doesn’t look like `part_of` is ever required by any biosample-derived type, so this might crash in that case, maybe?